src/store/modules/principal.js

```diff
@@ -103,6 +110,16 @@ const mutations = {
       Vue.set(unit, 'resources', [])
     }
   },
+  [M.OPEN_CONFIRMATION_DIALOG] (state, unitName) {
+    let unit = state.units.find(u => u.name === unitName);
+    if(unit){
+      state.activeUnit = unit;
+    }
+    state.confirmationDialogIsOpen = true;
```

Comment:
I'm not sure . Shouldn't we set confirmationDialogIsOpen  on true only if we found an unit ? ( in the if above)

Reply:
mmm. yes. Copy-paste from the existing code though ... 🥇